.github/workflows/nightly-builds.yml

```diff
@@ -120,12 +120,14 @@ jobs:
         # binary version is required and Akka build will set the right
         # full version from it.
         scalaVersion: ["2.12", "2.13"]
-        jdkVersion: ["adopt@1.8.0", "adopt@1.11"]
+        jdkVersion: ["adopt@1.8.0", "adopt@1.11", "openjdk@1.17.0"]
```

Comment:
I thought we had to open up some stuff?

Reply:
I think so too, but wanted to see Java tell me that to be sure.

Reply:
(I've been building on JDK 17 locally for a while but I guess the open up part will be needed to actually run anything)

Reply:
Minimal open-ups added to the build now.